cmd/apps/openfaas_app.go

```diff
@@ -154,32 +153,18 @@ func MakeInstallOpenFaaS() *cobra.Command {
 		}
 
 		createOperator, _ := command.Flags().GetBool("operator")
-		createOperatorVal := "false"
-		if createOperator {
-			createOperatorVal = "true"
-		}
-
 		clusterRole, _ := command.Flags().GetBool("clusterrole")
-
-		clusterRoleVal := "false"
-		if clusterRole {
-			clusterRoleVal = "true"
-		}
-
 		directFunctions, _ := command.Flags().GetBool("direct-functions")
-		directFunctionsVal := "true"
```

Comment:
Are you sure that the behaviour replaced here is the same as before?

Reply:
fairly sure - the command printed to console

```sh 
Command: /home/heyal/.arkade/bin/helm [upgrade --install openfaas openfaas/openfaas --namespace openfaas --wait --values /tmp/charts/openfaas/values.yaml --set openfaasImagePullPolicy=IfNotPresent --set faasnetes.imagePullPolicy=Always --set basicAuthPlugin.replicas=1 --set ingressOperator.create=false --set queueWorker.maxInflight=1 --set clusterRole=false --set gateway.directFunctions=true --set operator.create=false --set gateway.replicas=1 --set queueWorker.replicas=1 --set basic_auth=true --set serviceType=NodePort]

```

and the raw map printed 
```sh 
map[basicAuthPlugin.replicas:1 basic_auth:true clusterRole:false faasnetes.imagePullPolicy:Always gateway.directFunctions:true gateway.replicas:1 ingressOperator.create:false openfaasImagePullPolicy:IfNotPresent operator.create:false queueWorker.maxInflight:1 queueWorker.replicas:1 serviceType:NodePort]

```